int32 height 		# 1 = "high", 2 = "mid", 3 = "low"
---
